alkira/resource_alkira_service_infoblox_helper.go

```diff
@@ -64,7 +64,6 @@ func deflateInfobloxInstances(c []alkira.InfobloxInstance) []map[string]interfac
 			"anycast_enabled": v.AnyCastEnabled,
 			"hostname":        v.HostName,
 			"model":           v.Model,
-			"name":            v.Name,
```

Comment:
name was removed? This is on purpose?

Reply:
Yes. See discussion in terraform review channel. 

https://alkiranet.slack.com/archives/C038EL16SEA/p1653397061690509